lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -101,7 +101,8 @@ def self.framework
               when 4..6
                 :rails_notifications
               else
-                ::NewRelic::Agent.logger.error "Detected unsupported Rails version #{Rails::VERSION::STRING}"
+                ::NewRelic::Agent.logger.warning "Detected untested Rails version #{Rails::VERSION::STRING}"
+                :rails_notifications
```

Comment:
So in the end I think this is the best way to go. Simply log a warning if Rails is newer than either of the tested versions.

Reply:
I agree.